test/showroll_skeleton.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"math/rand"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+// don't have a sweetiebot instance
+// (or even go binary :P)
+// testing this on https://go-sandbox.com/
```

Comment:
This should just be noted in the commit or PR, and not in the code.

Reply:
The dangers of `git add .`